SU-HSDO/suhumsci
# shs-6: update breakout box background color

## Summary
This updates the breakout box background color per ticket request.

## Urgency
_'low'_

## Screenshots
![Screen Shot 2022-05-05 at 10 46 47 AM](https://user-images.githubusercontent.com/8405274/166951019-10dcc5f9-1b28-4a23-9dfe-9922fa67e472.png)

![Screen Shot 2022-05-05 at 10 47 12 AM](https://user-images.githubusercontent.com/8405274/166951037-f04cdf0c-1748-4173-8b9b-f9d3ff62bf1f.png)

![Screen Shot 2022-05-05 at 10 50 00 AM](https://user-images.githubusercontent.com/8405274/166951051-1793f316-cd12-4402-8c3b-7f57d8ce89a9.png)

![Screen Shot 2022-05-05 at 10 50 33 AM](https://user-images.githubusercontent.com/8405274/166951068-1948d360-0bdd-419d-8657-9ce37e7019aa.png)

## Steps to Test
1. Create a flexible page. Go to the WYSIWYG and add some text. Select the text and add the 'Breakout box' style as seen in the screenshot above.
2. Save the page and verify it's using the 'spotlight' color variable as the `background-color`. I included screenshots to demonstrate it passing accessibility, but feel free to verify yourself if you so choose.
